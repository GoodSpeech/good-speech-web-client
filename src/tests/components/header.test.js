import Header from '../../components/header';
import { render } from '../test-utils';

describe('Header component', () => {
  it('Should be rendered as expected', () => {
    const component = render(Header);
    expect(component.toJSON()).toMatchSnapshot();  
  });
});