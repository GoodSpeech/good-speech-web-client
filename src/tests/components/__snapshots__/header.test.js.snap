// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should be rendered as expected 1`] = `
<header
  className="MuiPaper-root-2758040674 MuiPaper-shadow4-941451405 MuiAppBar-root-2668177540 MuiAppBar-positionStatic-30597966 MuiAppBar-colorPrimary-15947162"
>
  <div
    className="MuiToolbar-root-1760419744 MuiToolbar-gutters-344431791"
  >
    <h2
      className="MuiTypography-root-233331062 MuiTypography-title-2106356098 MuiTypography-colorInherit-1808780954"
    >
      <img
        alt="good speech icon"
        className="Header-logo-1434838936"
        src="/icon.svg"
      />
      Good Speech
    </h2>
  </div>
</header>
`;
